tests/FSharp.Compiler.ComponentTests/CompilerCompatibilityTests.fs

```diff
@@ -0,0 +1,113 @@
+module FSharp.Compiler.ComponentTests.CompilerCompatibilityTests
+
+open System
+open System.IO
+open Xunit
+open FSharp.Test
+open FSharp.Test.Assert
+open TestFramework
+
+type CompilerCompatibilityTests() =
+
+    let projectsPath = Path.GetFullPath(Path.Combine(__SOURCE_DIRECTORY__, "../projects/CompilerCompat"))
+    let libProjectPath = Path.Combine(projectsPath, "CompilerCompatLib")
+    let appProjectPath = Path.Combine(projectsPath, "CompilerCompatApp")
+    
+    let runDotnetBuild projectPath useLocalCompiler =
```

Comment:
Extend this by also emitting (and then cleaning at the cleanup command) a `global.json` file which will enforce a hardcoded version of .NET (e.g. 9)